www/index.php

```diff
@@ -237,68 +244,25 @@
 				</div>
 			</div>
 			
-			<div id="setRootNode" class="fieldgroup" style="">
+			<div id="setRootNode" class="fieldgroup" style="display: none;">
```

Comment:
There are two kinds of ids used, one which is setRootNode and the other expressionContainer. Wouldn't it be better if we can follow one way of naming these things, and moreover the name of ids dont usually start with verbs as far as I think. It is an element and should be something more of a noun. What do you think? I think same goes for markRootNode.